bot/cogs/wolfram.py

```diff
@@ -65,14 +54,10 @@
 
 def custom_cooldown(*ignore: List[int]) -> check:
```

Comment:
Not your code, but this return type annotation is wrong. We call `check` which returns a `Callable`

**Note:** This means we should add `Callable` to the `from typing import` list.
```suggestion
def custom_cooldown(*ignore: List[int]) -> Callable:
```